playbooks/aws/openshift-cluster/build_ami.yml

```diff
@@ -39,6 +39,7 @@
 - import_playbook: ../../openshift-node/private/image_prep.yml
   vars:
     openshift_node_reboot_instance_before_cleanup: true
+    openshift_aws_ami_build_set_gquota_on_slashfs: false
```

Comment:
This would be controlled potentially by an inventory variable.  It's currently not necessary to use these plays with extra_vars.  Don't put this here, put it in role defaults.

Reply:
The value would have to be set in extra_vars.

Unfortunately we can't move it to a role var as this is a play that imports a play that imports another play.  This is the ugly of the aws plays and needs rearchitecting but not now/here.